packages/cli/src/api/e2e.api.ts

```diff
@@ -0,0 +1,107 @@
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable @typescript-eslint/no-unsafe-argument */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable @typescript-eslint/naming-convention */
+import { Router } from 'express';
+import bodyParser from 'body-parser';
+import { v4 as uuid } from 'uuid';
+import config from '@/config';
+import * as Db from '@/Db';
+import { RoleNames, RoleScopes } from '@/databases/entities/Role';
+import { hashPassword } from '@/UserManagement/UserManagementHelper';
+
+if (process.env.E2E_TESTS !== 'true') {
+	console.error('E2E endpoints only allowed during E2E tests');
+	process.exit(1);
+}
+
+const tablesToTruncate = [
+	'shared_workflow',
+	'shared_credentials',
+	'webhook_entity',
+	'workflows_tags',
+	'credentials_entity',
+	'tag_entity',
+	'workflow_entity',
+	'execution_entity',
+	'settings',
+	'installed_packages',
+	'installed_nodes',
+	'user',
+	'role',
+];
+
+const truncateAll = async () => {
+	const { connection } = Db;
+	for (const table of tablesToTruncate) {
+		await connection.query(
+			`DELETE FROM ${table}; DELETE FROM sqlite_sequence WHERE name=${table};`,
+		);
+	}
+	config.set('userManagement.isInstanceOwnerSetUp', false);
+};
+
+const setupUserManagement = async () => {
+	const { connection } = Db;
+	await connection.query('INSERT INTO role (name, scope) VALUES ("owner", "global");');
+	const instanceOwnerRole = (await connection.query(
+		'SELECT last_insert_rowid() as insertId',
+	)) as Array<{ insertId: number }>;
+
+	const roles: Array<[RoleNames, RoleScopes]> = [
+		['member', 'global'],
+		['owner', 'workflow'],
+		['owner', 'credential'],
+		['user', 'credential'],
+		['editor', 'workflow'],
+	];
```

Comment:
Ideally all six roles and the ID as part of the tuple. The part using `last_insert_rowid` comes the migrations, but I wonder in which case do roles not all have the same sequential IDs?

Reply:
`DELETE FROM sqlite_sequence` isn't working for some reason, so we can't simply use `1` for the instance-owner's role id.